File: 백준/Bronze/25304. 영수증/영수증.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner num = new Scanner(System.in);

        int X = num.nextInt();
        int N = num.nextInt();
        int sum = 0;

        for(int i = 0; i<N; i++) {
            int a = num.nextInt();
            int b = num.nextInt();
            sum += a*b;
        }
        if(sum==X) {
            System.out.println("Yes");
        }
        else {
            System.out.println("No");
        }
    }
}

File: 백준/Bronze/25304. 영수증/영수증.cc
#include <stdio.h>

int main(void) {
	int X, N, a, b, i, sum=0;
	scanf("%d %d", &X, &N);
	for(i=0; i<N; i++) {
		scanf("%d %d", &a, &b);
		sum+=a*b;
	}
	if(sum==X) {
		printf("Yes");
	} else {
		printf("No");
	}
	return 0;
} 